libs/libcommon/src/libcommon/queue.py

```diff
@@ -545,18 +545,18 @@ def create_jobs(self, job_infos: list[JobInfo]) -> int:
             return 0
 
     def delete_jobs_by_job_id(self, job_ids: list[str]) -> int:
```

Comment:
maybe change the name to `delete_waiting_jobs_by_job_id` to be more explicit

Reply:
done in [dbc2f25](https://github.com/huggingface/datasets-server/pull/2403/commits/dbc2f25763674ece0be3513f221d8d58b5767626)